src/components/vue-form-builder.vue

```diff
@@ -36,7 +36,7 @@
     </b-col>
 
     <!-- Renderer -->
-    <b-col class="overflow-auto mh-100 pl-4 pr-4 d-flex flex-column">
+    <b-col class="overflow-auto mh-100 pl-4 pr-4 d-flex flex-column position-relative">
```

Comment:
If you remove `pl-4 pr-4` and replace it with `ml-4 mr-4 p-0`, you can use `w-100` on the "drag an element here" container instead of the custom `width: calc(100% - 48px)`.